.github/workflows/feature_branch_ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Mifos-Mobile CI[Feature]
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!development'
+      - '!master'
+
+jobs:
+  build:
+    name: Build APK
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+        # Set up JDK
+      - name: Set Up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+        # Install NDK
```

Comment:
GitHub Actions require NDK and we are installing the same version of NDK as specified in `build.gradle` to avoid mismatches in NDK Version.
We need to update Gradle Permission because gradle files need to be executable ref. [link](https://github.com/actions/starter-workflows/issues/171)